e2e/pages/app/app.module.ts

```diff
@@ -5,219 +5,232 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 
 const routes: Routes = [
+    {
+        path: 'badge',
```

Comment:
I noticed that a custom prettier config had been added. We need the whole team to be using the same formatting settings, otherwise code is going to get rearranged with every commit, so preferably remove the custom config and update .editorconfig as suggested.

Reply:
Removed prettier which is causing the issues. This should hopefully not be a problem anymore.